app/assets/javascripts/hydra-editor/manage_repeating_fields.js

```diff
@@ -8,8 +8,8 @@ var HydraEditor = (function($) {
           this.options = options;
 
           this.controls = $("<span class=\"input-group-btn field-controls\">");
-          this.remover  = $("<button class=\"btn btn-danger remove\"><i class=\"icon-white glyphicon-minus\"></i><span>Remove</span></button>");
-          this.adder    = $("<button class=\"btn btn-success add\"><i class=\"icon-white glyphicon-plus\"></i><span>Add</span></button>");
+          this.remover  = $("<button type=\"button\" class=\"btn btn-danger remove\"><i class=\"icon-white glyphicon-minus\"></i><span>Remove</span></button>");
```

Comment:
Type is a valid attribute of <button> in HTML 5 but "The button has no default behavior." If scripting is the only way this button is useful, we *could* be preventing people who use assistive devices from using the form.